### PR TITLE
Profile (Settings) – Visual Design QA #396

### PR DESCRIPTION
**Pull request by megganeturner**
Addresses feedback in issue #396:
- labels are now semantic `<label>` tags, where they were a mix of heading and p tags. styles in line with invision spec
- updated the global `.btn-secondary` class to have inherit background-color
- custom down arrow for the select input, however as discussed the work for the custom dropdown menu will be done separately, as I suspect it would be a rather large chunk of work (:

Screen Shot 2021-02-05 at 4 37 10 pm

### CODE REVIEW

File: src/riot/Components/LanguageSelector.riot.html [hunk 1]
```diff
@@ -1,10 +1,12 @@
 <LanguageSelector>
-    <h4>{ TRANSLATIONS.language() }</h4>
+    <label for="language">{ TRANSLATIONS.language() }</label>
```
Comment (PeteCoward) [inline src/riot/Components/LanguageSelector.riot.html:2]: suggest we change this to "Select your language" to be more consistent with design

Comment (PeteCoward) [reply src/riot/Components/LanguageSelector.riot.html:2]: and it's a chance for @megganeturner to learn about our js translations framework.
- update the english version lower in this file
- run `yarn translations`
- ask someone who speaks tetum to provide a translation ( we usually do this all at once at the end )

**Review by katevoss — COMMENTED** [review]
This looks good to me, though there may be further design tweaks. A note about the font size is that the Canoe DS often calls for font size 12px which is quite small for mobile devices:

The [general](https://uxdesign.cc/guide-for-designing-better-mobile-apps-typography-5796495ef86f) [recommendation](https://www.smashingmagazine.com/2011/10/16-pixels-body-copy-anything-less-costly-mistake/) is that you shouldn't use anything smaller than 16px @andrewwapling

**Review by PeteCoward — APPROVED** [review]

**Comment by andrewwapling** [comment]
Hi @megganeturner @katevoss , 

The refinements are looking good. Please see additional commentary on visual review issue: https://github.com/catalpainternational/canoe/issues/396

Hope this is helpful and drop me a line if you have any queries at all.

Thanks again, Andy

**Pull request merged**